sourmash_lib/__init__.py

```diff
@@ -65,16 +74,32 @@ def is_molecule_type(self, molecule):
         return False
 
     def __getstate__(self):             # enable pickling
-        return (self.num, self.ksize, self.is_protein, self.mh.get_mins(),
-                self.hll, self.track_abundance)
+        with_abundance = False
+        if self.track_abundance:
+            with_abundance = True
+
+        return (self.num, self.ksize, self.is_protein,
+                self.mh.get_mins(with_abundance=with_abundance),
+                self.hll, self.track_abundance, self.max_hash,
+                self.seed)
 
     def __setstate__(self, tup):
         from . import _minhash
```

Comment:
Would the import of `_minhash` at the module level (suggested in a comment above) make this unnecessary ?